tools/amplify-preview/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/gravitational/shared-workflows/tools/amplify-preview
+
+go 1.23.3
+
+require (
+	github.com/alecthomas/kingpin/v2 v2.4.0
+	github.com/aws/aws-sdk-go v1.55.5
+	github.com/aws/aws-sdk-go-v2 v1.32.6
+	github.com/aws/aws-sdk-go-v2/config v1.28.6
+	github.com/aws/aws-sdk-go-v2/service/amplify v1.27.5
+	github.com/gravitational/shared-workflows/libs v0.1.3
+)
+
+require (
+	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137 // indirect
+	github.com/aws/aws-sdk-go-v2/credentials v1.17.47 // indirect
+	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.16.21 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/configsources v1.3.25 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.6.25 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/ini v1.8.1 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.12.1 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.12.6 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sso v1.24.7 // indirect
+	github.com/aws/aws-sdk-go-v2/service/ssooidc v1.28.6 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sts v1.33.2 // indirect
+	github.com/aws/smithy-go v1.22.1 // indirect
+	github.com/cli/go-gh/v2 v2.9.0 // indirect
+	github.com/cli/safeexec v1.0.0 // indirect
+	github.com/google/go-github/v63 v63.0.0 // indirect
+	github.com/google/go-querystring v1.1.0 // indirect
+	github.com/gravitational/trace v1.4.0 // indirect
+	github.com/xhit/go-str2duration/v2 v2.1.0 // indirect
+	golang.org/x/net v0.23.0 // indirect
+	golang.org/x/oauth2 v0.21.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+replace github.com/gravitational/shared-workflows/libs => ../../libs/
```

Comment:
This is needed to use new functions added to the `github` package.
Can be removed once new version is released.